Cesium3DTilesContent/include/Cesium3DTilesContent/ImplicitTilingUtilities.h

```diff
@@ -0,0 +1,346 @@
+   */
+  static CesiumGeometry::QuadtreeTileID getSubtreeRootID(
+      uint32_t subtreeLevels,
+      const CesiumGeometry::QuadtreeTileID& tileID) noexcept;
+
+  /**
+   * @brief Gets the ID of the root tile of the subtree that contains a given
+   * tile.
+   *
+   * @param subtreeLevels The number of levels in each sub-tree. For example, if
+   * this parameter is 4, then the first subtree starts at level 0 and
+   * contains tiles in levels 0 through 3, and the next subtree starts at
+   * level 4 and contains tiles in levels 4 through 7.
+   * @param tileID The tile ID for each to find the subtree root.
+   * @return The ID of the root tile of the subtree.
+   */
+  static CesiumGeometry::OctreeTileID getSubtreeRootID(
+      uint32_t subtreeLevels,
+      const CesiumGeometry::OctreeTileID& tileID) noexcept;
+
+  /**
+   * @brief Converts an absolute tile ID to a tile ID relative to a given root
+   * tile.
+   *
+   * For example, if `rootID` and `tileID` are the same, this method returns
+   * `QuadtreeTileID(0, 0, 0)`.
+   *
+   * @param rootID The ID of the root tile that the returned ID should be
+   * relative to.
+   * @param tileID The absolute ID of the tile to compute a relative ID for.
+   * @return The relative tile ID.
+   */
+  static CesiumGeometry::QuadtreeTileID absoluteTileIDToRelative(
+      const CesiumGeometry::QuadtreeTileID& rootID,
+      const CesiumGeometry::QuadtreeTileID& tileID) noexcept;
+
+  /**
+   * @brief Converts an absolute tile ID to a tile ID relative to a given root
+   * tile.
+   *
+   * For example, if `rootID` and `tileID` are the same, this method returns
+   * `OctreeTileID(0, 0, 0, 0)`.
+   *
+   * @param rootID The ID of the root tile that the returned ID should be
+   * relative to.
+   * @param tileID The absolute ID of the tile to compute a relative ID for.
+   * @return The relative tile ID.
+   */
+  static CesiumGeometry::OctreeTileID absoluteTileIDToRelative(
+      const CesiumGeometry::OctreeTileID& rootID,
+      const CesiumGeometry::OctreeTileID& tileID) noexcept;
+
+  /**
+   * @brief Gets a lightweight virtual container for enumerating the quadtree
+   * IDs of the children of a given quadtree tile.
+   *
+   * @param tileID The tile ID of the parent tile for which to get children.
+   * @return The children.
+   */
+  static QuadtreeChildren
+  getChildren(const CesiumGeometry::QuadtreeTileID& tileID) noexcept {
+    return QuadtreeChildren{tileID};
+  }
+
+  /**
+   * @brief Gets a lightweight virtual container for enumerating the octree
+   * IDs of the children of a given octree tile.
+   *
+   * @param tileID The tile ID of the parent tile for which to get children.
+   * @return The children.
+   */
+  static OctreeChildren
+  getChildren(const CesiumGeometry::OctreeTileID& tileID) noexcept {
+    return OctreeChildren{tileID};
+  }
+
+  /**
+   * @brief Computes the bounding volume for an implicit tile with the given ID.
```

Comment:
I think these comments could use a few extra words so it's easier to distinguish the functions at a glance. I'll add some changes in a commit.